plugin/src/main/java/com/spotify/plugin/dockerfile/AbstractDockerMojo.java

```diff
@@ -107,6 +107,12 @@ public String getFileName() {
   @Parameter(defaultValue = "false", property = "dockerfile.useMavenSettingsForAuth")
   protected boolean useMavenSettingsForAuth;
 
+  /**
+   * Whether to connect to Docker Daemon using HTTP proxy, if set.
+   */
+  @Parameter(defaultValue = "false", property = "dockerfile.useProxy")
```

Comment:
I think the current behavior of the DefaultDockerClient used in the plugin is to default to the proxy behavior being enabled. This would change the plugin to default to not enabling proxy behavior.

I think it would be better if the defaultValue for the parameter was `true` to keep current behavior the same - what do you think?